tests/mocks/translations.ts

```diff
@@ -1,15 +1,35 @@
 import merge from 'lodash/merge';
 
-const mockTranslation = (overrides = {}) =>
+export const mockTranslationKhattab = (overrides = {}) =>
   merge(
     {
-      id: 722482,
       languageName: 'english',
+      arabic: 'بِسْمِ ٱللَّهِ ٱلرَّحْمَـٰنِ ٱلرَّحِيمِ',
       text: 'In the Name of Allah—the Most Compassionate, Most Merciful.',
       resourceName: 'Dr. Mustafa Khattab',
-      resourceId: 102,
```

Comment:
Nit-picking. Let's revert this back. We should keep the mocked data as close to the actual API response as possible.